guava/src/main/java/com/fasterxml/jackson/datatype/guava/ser/MultimapSerializer.java

```diff
@@ -422,7 +420,26 @@ public void acceptJsonFormatVisitor(
     /* Internal helper methods
     /**********************************************************
      */
-    
+
+    /**
+     * @since 2.15
+     */
+    protected Multimap<?,?> _orderEntriesByKey(Multimap<?,?> value, JsonGenerator gen, SerializerProvider provider) throws IOException {
+        try {
+            if (value.containsKey(null)) {
+                return value;
+            }
+        } catch (NullPointerException e) {
+            return value;
+        }
+        try {
+            return TreeMultimap.create((Multimap<? extends Comparable, ? extends Comparable>) value);
+        } catch (ClassCastException e) {
+            // Neither key or value seems to be an instance of comparable
+            return value;
+        }
+    }
+
```

Comment:
Ok, on first part: unless you have observed actual issue, let's just not check for `null`s -- like you say, official implementations do not support `null` keys, so no point in trying to handle something that probably doesn't exist.

Reply:
Hmmh. Ok, true enough.

Reply:
@JooHyukKim Ok. So, `_sortKeys` is only set if there is explicit `@JsonFormat` on property, so that's where dev indicates sorting must be used. This is why my thinking was to explicitly fail if sorting is attempted and key type is not `Comparable`; and conversely, if flag not set, ignore `SerializationFeature` (can not compare if... not Comparable).

Reply:
Ok. I think I am ok with this as-is, then. We can consider additional problem if they occur.